src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
+  .slider {
+    display: none;
+  }
+}
+
+.bgimg {
+  background: url('../css/img/cv.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
Can use shorthand to merge most background attributes into one:

```css
background: url('../css/img/cv.jpg') fixed center no-repeat;
background-size: cover;
```

Syntax Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/background